commit bb6fdb523b3d750777fdeb9171658ea8ee080acb
Author: Than Ngo <than@redhat.com>
Date:   Tue Nov 28 00:53:03 2017 +0100

    set the submenu of "Manual Settings" insensitive if
    "Automated kdump memory settings" is enabled.

diff --git a/src/system-config-kdump.py b/src/system-config-kdump.py
index 2f96fca..3b5b4f7 100755
--- a/src/system-config-kdump.py
+++ b/src/system-config-kdump.py
@@ -1291,6 +1291,8 @@ class MainWindow:
         """
         self.manualdump_radiobutton.set_sensitive(state)
         self.manualdump_radiobutton.set_active(not state)
+        if self.my_settings.use_auto:
+           state = False
         self.total_mem_label.set_sensitive(state)
         self.kdump_mem_current_label.set_sensitive(state)
         self.kdump_mem_spin_button.set_sensitive(state)
